src/datumaro/plugins/yolo_format/converter.py

```diff
@@ -332,14 +340,26 @@ def _make_image_subset_folder(save_dir: str, subset: str) -> str:
     def _make_annotation_subset_folder(save_dir: str, subset: str) -> str:
         return osp.join(save_dir, YoloUltralyticsPath.LABELS_FOLDER_NAME, subset)
 
+    def _make_track_id_suffix(self, anno: Annotation):
+        track_id = anno.attributes.get("track_id") if self._write_track_id else None
```

Comment:
Shouldn't track_id attribute be checked to be int or converted to int? Probably, in all other formats we consider this to actually be a track id only if it's an int.

Reply:
checking it now (in the next line)